Cql/PackagerCLI/Program.cs

```diff
@@ -139,15 +139,19 @@ private static int ShowHelp()
                 """
                 Packager CLI Usage:
                 
-                    -?|-h|-help                   Show this help
+                    -?|-h|-help                     Show this help
                 
-                    --elm    <directory>          Library root directory
-                    --cql    <directory>          CQL root directory
-                    [--fhir] <file|directory>     Resource location, either file name or directory
-                    [--cs]   <file|directory>     C# output location, either file name or directory
-                    [--d]    <true|false>         Produce as a debug assembly
-                    [--f]    <true|false>         Force an overwrite even if the output file already exists
-                    [--canonical-root-url] <url>  The root url used for the resource canonical. If omitted a '#' will be used
+                    --elm       <directory>         Library root directory
+                    --cql       <directory>         CQL root directory
+                    [--fhir]    <file|directory>    Resource location, either file name or directory
+                    [--cs]      <file|directory>    C# output location, either file name or directory
+                    [--d]       <true|false>        Produce as a debug assembly
+                    [--f]       <true|false>        Force an overwrite even if the output file already exists
+                    [--canonical-root-url] <url>    The root url used for the resource canonical. 
+                                                    If omitted a '#' will be used
+                    [--override-utc-date-time] <utc-date-time>

```

Comment:
Recommend clarifying the expected string format - presumably ISO8601.  An example in the help would be useful.

Reply:
This PR is not ready for review. I have one in the workings with an example in the help